go/internal/feast/featurestore.go

```diff
@@ -58,10 +55,14 @@ func NewFeatureStore(config *registry.RepoConfig, callback transformation.Transf
 	if err != nil {
 		return nil, err
 	}
-  sanitizedProjectName := strings.Replace(config.Project, "_", "-", -1)
-  productName := os.Getenv("PRODUCT")
-  endpoint := fmt.Sprintf("%s-transformations.%s.svc.cluster.local:80", sanitizedProjectName, productName)
-  transformationService, _ := transformation.NewGrpcTransformationService(config, endpoint)
+
+	var transformationService *transformation.GrpcTransformationService = nil
```

Comment:
Earlier logic was handling: Flag=False and Endpoint=Value, Its using Transformation service. Now it's using GOPY bindings. Was that the intention when the code was written previously?

Reply:
No, this version of the code is what should be done. If flag=false, we should be using GOPY bindings. Flag should control whether we use the transformations server or not.